src/components/Counter/Counter.jsx

```diff
@@ -1,11 +1,51 @@
-import { useEffect, useState } from 'react'
+import { useReducer, useState, useEffect } from 'react'
 
 const pinkRGB = `rgb(236, 72, 153)`
 
+const initailCount = 0
+
+function countReducer(count, action) {
+  switch (action.type) {
+    case 'increment': {
+      return count + 1
+    }
+    case 'decrement': {
+      return count - 1
+    }
+    case 'reset': {
+      return initailCount
```

Comment:
I almost commented noting to be careful of typos and then I realized you did the same typo in both places. haha! Welp, that works too but also a reminder to be careful of typos.